packaging/ubuntu-16.04/rules

```diff
@@ -119,8 +119,8 @@ override_dh_auto_build:
 	cd cmd && ( ./configure --prefix=/usr --libexecdir=/usr/lib/snapd $(VENDOR_ARGS))
 	$(MAKE) -C cmd all
 
-	# Generate the real systemd units out of the available templates
-	$(MAKE) -C data/systemd all
+	# Generate the real systemd/upstart config files
```

Comment:
@zyga but we don't generate any upstart files, do we? We generate a dbus service file that will auto-start. Or am I missing something?